discohorts/pipeline.py

```diff
@@ -97,13 +102,15 @@ def get_patient_to_work_dir(patients, work_dirs):
                         check_call(command)
```

Comment:
I agree that `dry_run` is confusing as is. I do think there's utility in terms of not even calling `ocaml`; I've used this version of `dry_run` in testing quite a bit. So maybe I'll make a `python_dry_run` and `ketrew_dry_run`?